subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -453,28 +481,18 @@ internal class DokkaSymbolVisitor(
     }
 
     private fun KtAnalysisSession.visitEnumEntrySymbol(
-        enumEntrySymbol: KtEnumEntrySymbol
+        enumEntrySymbol: KtEnumEntrySymbol, scope: DokkaScope
     ): DEnumEntry = withExceptionCatcher(enumEntrySymbol) {
         val dri = getDRIFromEnumEntry(enumEntrySymbol)
         val isExpect = false
 
-        val scope = enumEntrySymbol.getMemberScope()
```

Comment:
`getMemberScope` for an enum entry is removed in https://github.com/JetBrains/kotlin/commit/e72a38dc824b2d395157bc39b317102fbdb32265